### DISCUSSION TITLE
Where do I put image files for gallery and slider extensions?

### DISCUSSION THREAD

**Discussion by derkolarsky** [datenstrom/community#441] [Ask a question] [answered]
Hi everyone,

I'm trying to find a stupid simple flat file CMS and yellow seems to work pretty good for me. But: I downloaded the extensions **gallery** and **slider** but don't get them to work.

I put the .zip file into /system/extensions and reload the page so the extension seems to get unzipped… Do I need to activate these somewhere or are they meant to work out of the box?

When having my i.e. Portfolio page and having a file called page.md inside and I want to put a gallery inside I just entered:
`[gallery https://…/media/gallery/subfolder/somefilename.*jpg - 20%]` or
`[gallery https://…/media/gallery/subfolder/somefilename*.jpg - 20%]` or whatever I always get the message after saving that "Gallery does not exist".

Am I missing something here? How is the code entered correctly? I couldn't find out more…

Thanks in advance   
Ralf

**Comment by schulle4u** [comment, marked as answer]
Hello Ralf, 

Slider and gallery extensions use `imageDir` setting from your `system/settings/system.ini` file. By default it is located at `media/images/`. Therefore all filename patterns start from this folder, absolute URLs are not allowed. However you can create subfolders in your image directory. 
Please see README for comprehensive examples: https://github.com/annaesvensson/yellow-gallery

HTH
Steffen

**Comment by derkolarsky** [comment]
Ahh… Got it. I thought I could put some subfolders just into **media** but haven't seen that **image** is the "master" under which I could put subfolders – works now, thanks! :)
